### PR TITLE
Look up book title on export

### PR DESCRIPTION
**Pull request by edwoodward**

### CODE REVIEW

File: api/admin.py [hunk 1]
```diff
@@ -3,13 +3,18 @@
 
 from django.contrib import admin
 from .models import CustomizationRequest
+from wagtail.core.models import Page
 
 class CustomizationRequestResource(resources.ModelResource):
     class Meta:
         model = CustomizationRequest
         fields = ('id', 'email', 'created', 'complete', 'num_students', 'book', 'reason', 'modules')
```
Comment (mwharrison) [inline api/admin.py:11]: Does it work if you do `book__title` here?

Comment (mwharrison) [reply api/admin.py:11]: If not, I would suggest using the `ForeignKeyWidget` instead. Which I think would like something like this:

```
class CustomizationRequestResource(resources.ModelResource):
	book = fields.Field(
        column_name='book',
        attribute='book',
        widget=ForeignKeyWidget(Book, 'title'))

    class Meta:
        model = CustomizationRequest
        fields = ('id', 'email', 'created', 'complete', 'num_students', 'book', 'reason', 'modules')
```

**Review by mwharrison — CHANGES_REQUESTED** [review]

File: api/admin.py (hunk 1 above)
Comment (mwharrison) [reply api/admin.py:11]: here's the documentation for that: https://django-import-export.readthedocs.io/en/latest/api_widgets.html#import_export.widgets.ForeignKeyWidget

Comment (edwoodward) [reply api/admin.py:18]: Modified to use widget

**Review by mwharrison — APPROVED** [review]

**Pull request merged**